frontend/packages/topology/src/components/list-view/list-view-utils.tsx

```diff
@@ -3,11 +3,6 @@ import * as _ from 'lodash';
 import { K8sKind, modelFor } from '@console/internal/module/k8s';
 import { getResourceKind } from '../../utils/topology-utils';
 
-export const translationForResourceKind = {
-  // t('topology~Helm Release')
-  HelmRelease: `topology~Helm Release`,
-};
-
 export const labelForNodeKind = (kindString: string) => {
```

Comment:
We should not split `labelKeyForNodeKind` and `labelForNodeKind`, as you moved the labelKey function into console-shared please move this function as well.